tests/containers_test.go

```diff
@@ -107,9 +107,6 @@ func TestPulumiTemplateTests(t *testing.T) {
 	for _, test := range testCases {
 		test := test
 		t.Run(test.template, func(t *testing.T) {
-			// TODO: Not running these in parallel to help with disk space.
-			// https://github.com/pulumi/pulumi-docker-containers/issues/215
-			// t.Parallel()
```

Comment:
We're removing the TODO here, but not enabling running in parallel :thinking: was this an oversight or intentional?

Reply:
Hmmm a little weird, we use program test below, which calls t.Parallel().

I updated this so that we call t.Parallel() ourselves, and tell program test not to. This way the `pulumi new` command should be part of the parallel execution too.

Reply:
Ah I had missed that.  ProgramTest calling parallel itself always trips me up.